localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/integration_request.py

```diff
@@ -26,15 +30,47 @@ def __call__(
 
         if integration_type in (IntegrationType.AWS_PROXY, IntegrationType.HTTP_PROXY):
             # `PROXY` types cannot use integration mapping templates
-            # TODO: check if PROXY can still parameters mapping and substitution in URI for example?
+            # TODO: check if PROXY can still parameters mapping and substitution in URI for example? normally not
             # See
             return
```

Comment:
Question: Would there be an advantage to returning an `IntegrationRequest` already populated with the `InvocationRequest` value? Could that streamline the work in the `_proxy` handlers?

Reply:
Yeah, I wonder that as well. Should the `_proxy` just directly use the `InvocationRequest`, or should we just manually basically copy `InvocationRequest` into `IntegrationRequest`, where we could for example remove the `Host` header. I believe this could be useful and also make sure all Integrations uses use `IntegrationRequest`